packages/ui-shared/lib/composables/useVeeValidate.ts

```diff
@@ -0,0 +1,24 @@
+import { Ref } from 'vue'
+import { useField } from 'vee-validate'
+
+export function useBaseField({
+  name,
+  type = 'string',
+  validation,
+  initialValue
+}: {
+  name: string
+  type?: string
+  initialValue?: string | number
+  validation?: Ref<string>
+}) {
+  if (type === 'number') {
```

Comment:
Can we also do like `isNaN(Number(initialValue))` here so we don't have to explicitly have the type set as `number`?